Mathlib/NumberTheory/Divisors.lean

```diff
@@ -43,8 +45,12 @@ def divisors : Finset ℕ := {d ∈ Ico 1 (n + 1) | d ∣ n}
   As a special case, `properDivisors 0 = ∅`. -/
 def properDivisors : Finset ℕ := {d ∈ Ico 1 n | d ∣ n}
 
-/-- `divisorsAntidiagonal n` is the `Finset` of pairs `(x,y)` such that `x * y = n`.
-  As a special case, `divisorsAntidiagonal 0 = ∅`. -/
+/-- Pairs of divisors of a natural number as a finset.
+
+`n.divisorsAntidiagonal` is the finset of pairs `(a, b) : ℕ × ℕ` such that `a * b = n`.
+As a special case, `Nat.divisorsAntidiagonal 0 = ∅`.
```

Comment:
I know this was already here, but the "As a special case, ..." seems a strange way to phrase it. I liked what you said in the PR description better. So maybe this?
```suggestion
`n.divisorsAntidiagonal` is the finset of pairs `(a, b) : ℕ × ℕ` such that `a * b = n`.
By convention, we choose `Nat.divisorsAntidiagonal 0 = ∅`.
```